Source/FieldSolver/SpectralSolver/SpectralFieldData.cpp

```diff
@@ -129,14 +162,24 @@ SpectralFieldData::ForwardTransform( const MultiFab& mf,
             Array4<Real> tmp_arr = tmpRealField[mfi].array();
             ParallelFor( realspace_bx,
             [=] AMREX_GPU_DEVICE(int i, int j, int k) noexcept {
-                tmp_arr(i,j,k) = mf_arr(i,j,k,i_comp);
+                tmp_arr(i,j,k) = mf_arr(i,j,k,i_comp);                
             });
         }
 
         // Perform Fourier transform from `tmpRealField` to `tmpSpectralField`
 #ifdef AMREX_USE_GPU
         // Add cuFFT-specific code ; make sure that this is done on the same
         // GPU stream as the above copy
```

Comment:
Could you replace `Add cuFFT-specific code ` by `Perform FFT on GPU `?
Please keep the rest of the comment (`make sure that this is done on the same ...`), since it is helpful to someone who reads the code.

Reply:
Done.